CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -291,6 +336,22 @@ edm::ESProducts< boost::shared_ptr<TotemDAQMapping>, boost::shared_ptr<TotemAnal
 
 //----------------------------------------------------------------------------------------------------
 
+DOMDocument* TotemDAQMappingESSourceXML::GetDOMDocument(string file)
+{
+  XercesDOMParser* parser = new XercesDOMParser();
```

Comment:
The Initialize and Terminate calls are here:
https://github.com/CTPPS/cmssw/blob/ctpps_conf_daq_mapping/CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc#L313
https://github.com/CTPPS/cmssw/blob/ctpps_conf_daq_mapping/CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc#L331

The "parser" really doesn't seem to be deleted. I will fix it.